samples/readme.samples.md

```diff
@@ -4,4 +4,20 @@
 require: $(this-folder)/../readme.md
 output-folder: $(this-folder)/$(title)/$(title)
 clear-output-folder: false
+```
+
+``` yaml $(include-csproj) != 'disable'
+pipeline:
+  csharpproj:
+    input: modelerfour/identity
+  csharpproj/emitter:
+    input: csharpproj
+    scope: output-scope
+```
+
+``` yaml $(dotnet-build) != 'disable'
```

Comment:
Why don't we put this logic into a common location with test projects?

Reply:
Sure. I'll do that in a separate PR and just clean up all shared readmes (root, samples, TestProject, TestServerProjects). Give me a bit.